src/index.js

```diff
@@ -0,0 +1,14 @@
+// import _ from 'lodash';
+import './style.css';
+
+function component() {
+  const element = document.createElement('div');
```

Comment:
- [ ] In index.js you should create an array of some simple to do tasks (array of objects). Each task object should contain three keys:
```
description [string].
completed [bool].
index: [number].
```
- [ ] Write a function to iterate over the tasks array and populate an HTML list item element for each task. This should be done dynamically like we did with the project cards on our `Portfolio` projects
e.g
```
The implementation does not have to be exact

const populateTasks = (task, parentElement) => {
  const { description, completed, index: id } = task;
  const innerHTML = `
         <p>${description}</p>
         <p>${completed ? 'completed' : 'not completed'}</p>`;
  const newTodoListElement = document.createElement('li');
  Object.assign(newTodoListElement, { innerHTML, id });
  parentElement.appendChild(newTodoListElement);
};
```